### ISSUE TITLE
Feature: Toggle Button for Wider Text Layout (Both Web and Desktop App)

### ISSUE THREAD

**Issue by SalvaZaraes** [seed-hypermedia/seed#58]
Description:
Add a button to toggle a wider text layout (100 characters per line) when reading or editing a document. This would offer an alternative reading and editing experience, catering to users who prefer a wider text display.

**Comment by ericvicenti** [comment]
Yes it would be nice to have a "view settings" where the reader can change cosmetic settings for their own needs and preferences, including font size, wider text layout, dark/light mode, and more
